packages/syft/src/syft/client/client.py

```diff
@@ -86,6 +86,7 @@ def upgrade_tls(url: GridURL, response: Response) -> GridURL:
 
 
 def forward_message_to_proxy(
+    node_connection: NodeConnection,
```

Comment:
suggestion: Let's remove node_connection from `forward_message_to_proxy` method, since we don't use it.